src/libraries/System.Runtime/tests/System.IO.Tests/StreamReader/StreamReader.StringCtorTests.cs

```diff
@@ -19,10 +19,6 @@ public static void NullArgs_ThrowsArgumentNullException()
             AssertExtensions.Throws<ArgumentNullException>("path", () => new StreamReader((string)null, null, true));
             AssertExtensions.Throws<ArgumentNullException>("path", () => new StreamReader((string)null, null, true, null));
             AssertExtensions.Throws<ArgumentNullException>("path", () => new StreamReader((string)null, null, true, -1));
-            AssertExtensions.Throws<ArgumentNullException>("encoding", () => new StreamReader("path", (Encoding)null));
-            AssertExtensions.Throws<ArgumentNullException>("encoding", () => new StreamReader("path", null, true));
-            AssertExtensions.Throws<ArgumentNullException>("encoding", () => new StreamReader("path", null, true, null));
-            AssertExtensions.Throws<ArgumentNullException>("encoding", () => new StreamReader("path", null, true, -1));
```

Comment:
Do we have tests that pass in null successfully?

Reply:
@stephentoub 
You're right! I'm gong to add these tests that focusing on null value passing today.
Thanks!

Reply:
@stephentoub 
Done adding unit tests to test calling StreamReader's ctors with null Encoding param passed.
Please review, thanks in advance!

Reply:
refactor the test to only focus on testing ctors with Stream param instead of string path, as it always throw exceptions caused by checking actual path.

Please review as needed.